plugin/hpsa/hpsa.py

```diff
@@ -113,10 +113,27 @@ def _parse_hpssacli_output(output):
         cur_indent_count = len(cur_line) - len(cur_line.lstrip())
         nxt_indent_count = len(nxt_line) - len(nxt_line.lstrip())
 
+        if cur_indent_count == top_indention_level:
+            flag_required_section = True
+
+        if cur_indent_count == second_indention_level:
+            flag_required_section = False
+
+            if nxt_indent_count == cur_indent_count:
+                flag_required_section = True
+            else:
+                for required_section in required_sections:
+                    if cur_line.lstrip().startswith(required_section):
+                        flag_required_section = True
+
+        if flag_required_section is False:
+            continue
+
         cur_line_splitted = cur_line.split(": ")
         cur_data_pointer = indent_2_data[cur_indent_count]
 
         if nxt_indent_count > cur_indent_count:
+            # Current line is new section title
             nxt_line_splitted = nxt_line.split(": ")
```

Comment:
nxt_line_splitted is a rough variable name. nxt_line_split would be better, or even nxt_line_split_res (for next line split result, if you will).